package/package.json

```diff
@@ -2,8 +2,10 @@
 	"name": "janpio-exports-map-example",
 	"version": "0.0.3",
 	"description": "Isomorphic secure-random library, demonstrating `exports` use with Workers",
+  "main": "./src/node.js",
```

Comment:
(Err I used spaces instead of tabs)

I added this because it makes it closer to reality, that also shows that the folder projects can succeed (as they do with Prisma), and default to loading `main` and not the exports map.

That's an interesting learning in itself: Bundlers do read the package.json for `main`, `browser`, yet they ignore `exports`.

Reply:
Ah, that explains what that one suddenly succeeds.

Shouldn't this point to `index.js` then to show that it does something completely different than what it should be doing via the exports map?

Reply:
Yes, we could add an index.js that can show that behavior